src/repeated_game.jl

```diff
@@ -86,6 +100,15 @@ best_dev_payoff_2(rpd::RepGame2, a1::Int) =
 Places `npts` equally spaced points along the 2 dimensional circle and returns
 the points with x coordinates in first column and y coordinates in second
 column.
+
+# Arguments
+
+- `npts::Int` : Number of points to be placed.
+
+# Returns
+
+- `pts::Array{Float64}(npts, 2)` : The array containing the coordinates of the
```

Comment:
Should be something like ``- `H::Matrix{Float64}` : ... of shape (or size?) `(nH, 2)` ...``.